### PR TITLE
Include acbuild from master for new OCI build mode support.

### PR DESCRIPTION
**Pull request by daithiocrualaoich**

### CODE REVIEW

**Comment by passy** [comment]
Travis isn't happy with the GPG key import. Is this a transient issue?

File: dev/dev-rktmachine.build.sh [hunk 1]
```diff
@@ -63,6 +63,14 @@ acbuild run -- apt-get install -qq qemu
 
 
 ################################################################################
+# acbuild Master Build
+################################################################################
+
+acbuild run -- apt-get install -qq git
+acbuild run -- apt-get install -qq golang
```
Comment (passy) [inline dev/dev-rktmachine.build.sh:70]: Vile.

File: docs/building.rst [hunk 2]
```diff
@@ -160,6 +160,41 @@ Install the ``acbuild`` binaries by downloading them from the
     sudo tar xzvf acbuild-v0.4.0.tar.gz -C tools --strip-components=1
     sudo chmod u+s tools/acbuild
 
+Alternatively to build the latest ``acbuild`` from master instead, start the
+container as before.
+
+::
+
+    sudo rkt run \
```
Comment (passy) [inline docs/building.rst:168]: Random question: Is sudo-less rkt on the roadmap? I feel like I've read something about this in the past.

File: docs/building.rst [hunk 3]
```diff
@@ -99,7 +99,7 @@ Rebuilding ``tools.qcow2``
 The ``tools.qcow2`` image file contains binaries needed for VM setup which are
 too large to be delivered by Cloud Init. Specifically, it contains:
 
-- The `acbuild` binaries for building new container images on the CoreOS VM.
+- The ``acbuild`` binaries for building new container images on the CoreOS VM.
```
Comment (passy) [inline docs/building.rst:102]: Good catch!

**Review by passy — APPROVED** [review]
Good stuff!

File: docs/building.rst (hunk 3 above)
Comment (daithiocrualaoich) [reply docs/building.rst:102]: No, really. :) RestructuredText doesn't specify a formatting style for single backticks in renderers. They all probably do the same as double backticks but are you willing to risk it?

**Comment by daithiocrualaoich** [comment]
The Travis issue started in https://github.com/woofwoofinc/rktmachine/commit/a2197db6605c45bab54ad6b6a9adedc797fa2375 where we needed an updated Ruby for https://github.com/travis-ci/travis-ci/issues/6307 and it just keeps running.

**Comment by daithiocrualaoich** [comment]
91ee726ab48

You aren't going to like the fix...

File: docs/building.rst (hunk 2 above)
Comment (daithiocrualaoich) [reply docs/building.rst:168]: https://github.com/rkt/rkt/issues/1585

**Pull request merged**